cli.js

```diff
@@ -14,7 +14,11 @@ args.command(['search', 's'], 'Seach for plugins or themes.', (name, sub) => {
       spinner.color = 'green';
       spinner.text = chalk.green(`Results for "${searchTerm}":`);
       spinner.succeed();
-      console.log(results);
+      if(results.length > 0) {
+        results.forEach(result => console.log(' - ' + result));
```

Comment:
There are cleaner approach to this.

First: use reduce or map to build a list of strings. At least in the end, you can `console.log` __once__
Second: use string interpolation/templating coming with ES6

just an example

```js
const resultMessage = results.length
  ? results.reduce(result => `- ${result}`, '').join('\n')
  : 'no packages found'

console.log(chalk.yellow(resultMessage))
```